libs/domains/organizations/feature/src/lib/hooks/use-member-invitation/use-member-invitation.tsx

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseMemberInvitationProps {
+  organizationId?: string
+  inviteId?: string
+  enabled?: boolean
+}
+
+export function useMemberInvitation({ organizationId, inviteId, enabled = true }: UseMemberInvitationProps) {
+  return useQuery({
+    ...queries.organizations.memberInvitation({ organizationId: organizationId!, inviteId: inviteId! }),
+    enabled: !!organizationId || !!inviteId || enabled,
```

Comment:
Should be 
```suggestion
    enabled: !!organizationId && !!inviteId && enabled,
```
no?

Reply:
https://github.com/Qovery/console/pull/1119/commits/2c58392c2faef32eab45cc8d28b44127eed6a109